cmd/titus-vpc-tool/setup_container.go

```diff
@@ -15,17 +15,19 @@ func setupContainercommand(ctx context.Context, v *pkgviper.Viper, iipGetter ins
 		Use:   "setup-container",
 		Short: "Setup networking for a particular container",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			netns := v.GetInt("netns")
+			pid1dirfd := v.GetInt("pid-1-dir-fd")
+			transitionNamespaceDir := v.GetString("transition-namespace-dir")
 			switch strings.ToLower(v.GetString(generationFlagName)) {
 			case "v2", "v3":
-				return container2.SetupContainer(ctx, iipGetter(), netns)
+				return container2.SetupContainer(ctx, iipGetter(), pid1dirfd, transitionNamespaceDir)
 			default:
 				return fmt.Errorf("Version %q not recognized", v.GetString(generationFlagName))
 			}
 		},
 	}
 
-	cmd.Flags().Int("netns", 3, "The File Descriptor # of the network namespace to setup")
+	cmd.Flags().Int("pid-1-dir-fd", 3, "The File Descriptor # of the pid 1 directory to setup")
 	cmd.Flags().String(generationFlagName, generationDefaultValue, "Generation of VPC Tool to use, specify v1, or v2")
+	cmd.Flags().String("transition-namespace-dir", "/run/transition", "Directory to mount transition namespaces into")
```

Comment:
`Directory in the container to mount transition namespaces into` ?
Actually, is it in the container? 
Or is it not, just the fd gets bind-mounted?